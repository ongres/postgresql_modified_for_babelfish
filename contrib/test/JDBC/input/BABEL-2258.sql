select 'foo' where 'bar ' = 'bar';
go

select 'foo' where 0xAE = 0xAE00;
go

select 'foo' where 101.5E5 = 1015E4;
go

select 'foo' where (select 'bar ') = (select 'bar');
go