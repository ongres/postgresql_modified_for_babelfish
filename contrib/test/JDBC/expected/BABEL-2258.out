select 'foo' where 'bar ' = 'bar';
go
~~START~~
varchar
foo
~~END~~


select 'foo' where 0xAE = 0xAE00;
go
~~START~~
varchar
foo
~~END~~


select 'foo' where 101.5E5 = 1015E4;
go
~~START~~
varchar
foo
~~END~~


select 'foo' where (select 'bar ') = (select 'bar');
go
~~START~~
varchar
foo
~~END~~

